content/en/docs/reference/glossary/kubectl.md

```diff
@@ -4,16 +4,19 @@ id: kubectl
 date: 2018-04-12
 full_link: /docs/user-guide/kubectl-overview/
 short_description: >
-  A command line tool for communicating with a Kubernetes API server.
+  A command line tool for communicating with a Kubernetes cluster.
```

Comment:
A command line tool for communicating with a Kubernetes cluster API server.

Reply:
for me both is right depending on the target group

Reply:
Calling out, "Kubernetes API server" seems more specific than "cluster". Is it more important to identify the Kubernetes API server?

Reply:
Here's why I prefer “cluster”: if we say “API server” in the tooltip, I'm worried that some newcomers will feel that they need to learn what a Kubernetes API server is&mdash;not an easy thing to grasp&mdash;and that some proportion of those will just write off Kubernetes as too hard to learn.

The full description is welcome to make the details clear.